packages/mui-material/src/Menu/Menu.test.js

```diff
@@ -381,27 +375,20 @@ describe('<Menu />', () => {
     });
   });
 
-  describe('paper', () => {
-    it('should use MenuPaper component', () => {
-      const wrapper = mount(
-        <Menu anchorEl={document.createElement('div')} open>
-          <div />
-        </Menu>,
-      );
-
-      expect(wrapper.find(MenuPaper)).to.have.length(1);
-    });
-  });
```

Comment:
I think this test doesn't add value. It just tests implementation details.

Reply:
I agree we should remove this one. Do you see any other way of testing that we don't fall into the same issue again?, which is explained here: https://github.com/mui/material-ui/pull/37390

Reply:
We have a couple of [tests](https://github.com/mui/material-ui/blob/next/packages/mui-material/src/Menu/Menu.test.js#L310-L382) that check for the correct behavior when `MuiMenu.paper` (`MenuPaper` under the hood) styles are overridden.

If I understood the issue correctly, these two tests cover what we want to ensure: styles are correctly applied to `MuiMenu.paper`. Checking if a `MenuPaper` component is rendered is kinda pointless because it doesn't give us any valuable information without actually checking what `MenuPaper` does.